docker-compose.yml

```diff
@@ -27,6 +27,7 @@ services:
       FIDES__DEV_MODE: "True"
       FIDES__REDIS__ENABLED: "True"
       FIDES__USER__ANALYTICS_OPT_OUT: "True"
+      FIDES__SECURITY__ALLOW_CUSTOM_CONNECTOR_FUNCTIONS: "True"
```

Comment:
Defaulting to true since we'll probably want this to always be enabled for testing